ccloud-monitoring/start.sh

```diff
@@ -48,6 +48,9 @@ eval $CMD \
     && print_code_pass -c "source $DELTA_CONFIGS_ENV" \
     || exit_with_error -c $? -n "$NAME" -m "$CMD" -l $(($LINENO -3))
 
+printf "\n====== Allow service account to describe Confluent Cloud cluster and consumer groups\n"
```

Comment:
Thanks for identifying this ACL gap.  @awalther28 @rspurgeon my leaning is to move this into `ccloud-stack` creation (`utils/ccloud_library.sh`) for all other future use cases to benefit.  Thoughts?

Reply:
I think that would be ideal :) happy to move it into there in this PR.

Reply:
Yeah, let's do it.

Reply:
Moved, updated the start script and instructions. I validated the start script and tearing the cluster down. 
A few unrelated ACLs weren't being deleted in utils/ccloud_library.sh, so I added them to be deleted as well.